Secret-agency-service/src/test/java/cz/fi/muni/pa165/secretagency/service/facade/MissionFacadeTest.java

```diff
@@ -0,0 +1,128 @@
+package cz.fi.muni.pa165.secretagency.service.facade;
+
+import cz.fi.muni.pa165.secretagency.dto.MissionCreateDTO;
+import cz.fi.muni.pa165.secretagency.dto.MissionDTO;
+import cz.fi.muni.pa165.secretagency.entity.Mission;
+import cz.fi.muni.pa165.secretagency.enums.MissionTypeEnum;
+import cz.fi.muni.pa165.secretagency.facade.MissionFacade;
+import cz.fi.muni.pa165.secretagency.service.BeanMappingService;
+import cz.fi.muni.pa165.secretagency.service.MissionService;
+import cz.fi.muni.pa165.secretagency.service.config.ServiceConfiguration;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests for mission facade.
+ * @author Adam Skurla (487588)
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
```

Comment:
Pro jistotu bych přidal i anotaci DirtiesContext. Tak, jak je to u ostatních testovacích tříd.